### PR TITLE
Carletop settled mbta etc

### PR DESCRIPTION
**Pull request by carletop**
I believe we've already implicitly received the ok for all these changes through Matthew's latest comments to the inline documents. Please review and accept as appropriate. I've added more descriptive comments for most commits than is displayed in the commit title. I believe this branch merge should precede that with the latest commits.

### CODE REVIEW

File: spec/en/reference.md [hunk 1]
```diff
@@ -108,38 +120,53 @@ If an agency can associate supplementary ridership data with a specific rider, r
 |  Field Name | Required | Details |
 |  ------ | ------ | ------ |
 | rider_id | **Required** | The **rider_id** field contains the ID of a unique rider. The **rider_id** is dataset unique. |
-| trip_id | **Required** | The **trip_id** field contains the ID of the trip associated with the unique rider |
+| trip_id | Optional | The **trip_id** field contains the ID of the trip associated with the unique rider, if this is known. If **trip_id** is empty then the rider may have taken one of several trips, or several possible chains of trips, to travel between the origin and the destination.   |
 | boarding_stop_id | Optional | The **boarding_stop_id** field contains the ID of the boarding stop associated with the unique rider. |
+| boarding_stop_sequence | Optional | The **boarding_stop_sequence** field identifies the order of the stop referenced **boarding_stop_id** within a particular trip. Matches **stop_sequence** in _stop_times.txt_. Non-negative integer. |
```
Comment (hooversy) [inline spec/en/reference.md]: We should hyperlink to GTFS _stop_times.txt_.

File: spec/en/reference.md [hunk 2]
```diff
@@ -108,38 +120,53 @@ If an agency can associate supplementary ridership data with a specific rider, r
...
 | alighting_stop_id | Optional | The **alighting_stop_id** field contains the ID of the alighting stop associated with the unique rider. |
-| boarding_time | Optional | The **boarding_time** field contains the time of the boarding associated with the unique rider. |
-| boarding_date | Optional | The **boarding_date** field contains the date of the boarding associated with the unique rider. |
-| alighting_time | Optional | The **alighting_time** field contains the time of the alighting associated with the unique rider. |
-| alighting_date | Optional | The **alighting_date** field contains the date of the alighting associated with the unique rider. |
+| alighting_stop_sequence | Optional | The **alighting_stop_sequence** field identifies the order of the stop referenced **alighting_stop_id** within a particular trip. Matches **stop_sequence** in _stop_times.txt_. Non-negative integer. |
```
Comment (hooversy) [inline spec/en/reference.md]: We should hyperlink to GTFS _stop_times.txt_.

File: spec/en/reference.md [hunk 3]
```diff
@@ -108,38 +120,53 @@ If an agency can associate supplementary ridership data with a specific rider, r
...
+| service_date | Optional | The **service_date** field contains the date of the boarding associated with the unique rider. |
+| boarding_time | Optional | The **boarding_time** field contains the time of the boarding associated with the unique rider. If the **trip_id** is included, **boarding_time** represents the time that the rider boarded the vehicle. It must be between **service_arrival_time** and **service_departure_time**, inclusive, of the trip’s stop in _board_alight.txt_ if applicable. If the trip_id is not included, **boarding_time** represents the time that the rider entered the transit network. |
```
Comment (hooversy) [inline spec/en/reference.md]: We should hyperlink to _board_alight.txt_.

File: spec/en/reference.md [hunk 4]
```diff
@@ -108,38 +120,53 @@ If an agency can associate supplementary ridership data with a specific rider, r
...
+| alighting_time | Optional | The **alighting_time** field contains the time of the alighting associated with the unique rider. If the **trip_id** is included, **alighting_time** represents the time that the rider exited the vehicle. It must be between **service_arrival_time** and **service_departure_time**, inclusive, of the trip’s stop in _board_alight.txt_ if applicable. If the trip_id is not included, **alighting_time** represents the time that the rider left the transit network. |
```
Comment (hooversy) [inline spec/en/reference.md]: We should hyperlink to _board_alight.txt_.

File: spec/en/reference.md [hunk 5]
```diff
@@ -108,38 +120,53 @@ If an agency can associate supplementary ridership data with a specific rider, r
...
+|   |  | * **1** - customer used stored value or tokens.  |
+|   |  | * **2** - customer used a transfer.  |
+|   |  | * **3** - customer used a pass.  |
+|   |  | * **4** - customer used a promotional coupon.  |
+|   |  | * **5** - trip is a free trip.  |
+|   |  | * **6** - customer was never charged the fare.  |
+|   |  | * **7** - customer was charged a fare but did not pay.  |
+|   |  | * **8** - Other.  |
+| fare_media | Optional | The **fare_media** field indicates what media was used to pay the fare. |
+|   |  | * **0** - (or blank) Not applicable or unknown.  |
```
Comment (hooversy) [inline spec/en/reference.md]: Blank is redundant for unknown. Perhaps remove blank from option description.

File: spec/en/reference.md (hunk 1 above)
Comment (carletop) [inline spec/en/reference.md]: done in "Matthew" branch

File: spec/en/reference.md (hunk 2 above)
Comment (carletop) [inline spec/en/reference.md]: done in "Matthew" branch

File: spec/en/reference.md (hunk 3 above)
Comment (carletop) [inline spec/en/reference.md]: done in "Matthew" branch

File: spec/en/reference.md (hunk 4 above)
Comment (carletop) [inline spec/en/reference.md]: done in "Matthew" branch

File: spec/en/reference.md (hunk 5 above)
Comment (carletop) [inline spec/en/reference.md]: done

**Pull request merged**